server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const router = require('./routes.js');
+const helper = require('../db/index.js');
+const bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+app.use(express.static('../public'));
+
+app.get('/', (req, res) => {
```

Comment:
@waynekwest88 Unclear on why .get('/') is  necessary when serving static files from Public folder? Is this necessary or can it be removed?